cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,20 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
+                for vertical in item.get_children():
+                    group_id = int(vertical.display_name.split(" ")[2])
+                    groups_usage_info.update({
+                        unicode(get_item_location(vertical)): [group_id],
+                    })
+                    for component in vertical.get_children():
+                        groups_usage_info.update({
+                            unicode(get_item_location(component)): [group_id]
+                        })
+
             if groups_usage_info:
-                item_location = item.location.version_agnostic().replace(branch=None)
+                item_location = get_item_location(item)
```

Comment:
With the addition on the `get_item_location` method here, seems like we no longer really have a need for `item_location` variable